enterprise/frontend/src/embedding-sdk/cli/utils/get-component-snippets.ts

```diff
@@ -26,33 +35,44 @@ export function getComponentSnippets(options: Options) {
     ? ANALYTICS_PROVIDER_SNIPPET_WITH_USER_SWITCHER
     : ANALYTICS_PROVIDER_SNIPPET_MINIMAL;
 
-  const components = [
+  const components: SampleComponent[] = [
     {
-      name: "analytics-provider",
-      content: analyticsProviderSnippet.trim(),
+      fileName: "analytics-provider",
+      componentName: "AnalyticsProvider",
+      content: withNextJsDirective(analyticsProviderSnippet, isNextJs).trim(),
```

Comment:
We use a `useState` here to store the current theme to demo the theme switcher (dark/light), so we have to add a "use client" directive here.